test/integration/query/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+addtest(query_acceptance_test
+        query_acceptance_test.cpp
+        )
+target_link_libraries(query_acceptance_test
+        application
+        gtest
```

Comment:
It is linked with all of the test by default

Reply:
-

Reply:
--

Reply:
+